pkg/client/unversioned/discovery_client.go

```diff
@@ -154,7 +126,9 @@ func (d *DiscoveryClient) ServerResources() (map[string]*unversioned.APIResource
 
 func setDiscoveryDefaults(config *Config) error {
 	config.Prefix = ""
-	config.Version = ""
+	config.Version = "unversioned"
```

Comment:
The `Version` is not being used. I put it there to pass the check [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/client/unversioned/helper.go#L392). Is it ok?